README.md

```diff
@@ -119,61 +83,28 @@ this functionality you can follow the following steps.
 To enable third-party scripts that you've added you will need to add the package
```

Comment:
It's probably worth mentioning that it's easier (and avoids issues with valid JSON, double quotes, and commas) if you don't edit `package.json`, and simply use the command `npm install --user [package-name]`.